llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -3002,6 +3001,8 @@ void LoopAccessInfo::print(raw_ostream &OS, unsigned Depth) const {
 
   // List the pair of accesses need run-time checks to prove independence.
   PtrRtChecking->print(OS, Depth);
+  if (PtrRtChecking->Need && !CanVecMem)
```

Comment:
I think this check is a bit too lax, and generates spurious messages. Can you check CheckingGroups against DepCands instead?